src/vm/eetwain.cpp

```diff
@@ -3991,6 +3981,29 @@ bool EECodeManager::UnwindStackFrame(PREGDISPLAY     pContext,
     return true;
 }
 
+#endif // _TARGET_X86_
+
+#if defined(_TARGET_X86_) && !defined(WIN64EXCEPTIONS)
+
+/*****************************************************************************
+ *
+ *  Unwind the current stack frame, i.e. update the virtual register
+ *  set in pContext. This will be similar to the state after the function
+ *  returns back to caller (IP points to after the call, Frame and Stack
+ *  pointer has been reset, callee-saved registers restored (if UpdateAllRegs),
+ *  callee-unsaved registers are trashed.
+ *  Returns success of operation.
+ */
+
+bool EECodeManager::UnwindStackFrame(PREGDISPLAY     pContext,
+                                     EECodeInfo     *pCodeInfo,
+                                     unsigned        flags,
+                                     CodeManState   *pState,
+                                     StackwalkCacheUnwindInfo  *pUnwindInfo /* out-only, perf improvement */)
+{
+    return UnwindStackFrameEx(pContext, pCodeInfo, flags, pState, pUnwindInfo);
```

Comment:
The "Ex" seems little strange since the standalone function is not an extended version. You could have kept the same name and just classify it by "::" to indicate that the global version should be called here:
`return ::UnwindStackFrame(pContext, pCodeInfo, flags, pState, pUnwindInfo);`
Would you mind changing it that way?